includes/class-scriptlesssocialsharing-output.php

```diff
@@ -147,6 +147,23 @@ public function do_buttons( $output, $heading = true ) {
 	}
 
 	/**
+	 * Get the permalink to be shared via the button.
+	 *
+	 * @param  string $button_name The name of the button, e.g. 'twitter', 'facebook'.
+	 * @return string The URL to be shared.
+	 */
+	protected function get_permalink( string $button_name ) {
+		$attributes = $this->attributes();
+		return rawurlencode(
```

Comment:
Would it maybe make more sense to use the `rawurlencode` function on the final URL output? See [line 140](https://github.com/robincornett/scriptless-social-sharing/blob/develop/includes/class-scriptlesssocialsharing-output.php#L140)?